ensime_shared/protocol.py

```diff
@@ -94,6 +97,12 @@ def handle_type_inspect(self, call_id, payload):
     def show_type(self, call_id, payload):
         raise NotImplementedError()
 
+    def handle_source_positions(self, call_id, payload):
+        raise NotImplementedError()
```

Comment:
Sorry, I'm not following. This is the ugly way for defining an abstract method in python, so it doesn't need to be filled here.

Anyway, after @ches finishes his big refactor I'll rework the handlers, I think resolving them for the server version using inheritance is a terrible design and I want to change it. Also they should depend on protocol version as opposed to on server version.